### PR TITLE
Dec 15: Add pitch detection post

### PR DESCRIPTION
**Pull request by jjatria**
The date is tentative. This could go earlier if proofread and needed

### CODE REVIEW

File: statocles-site/blog/2024/12/16/pitch-detection/index.markdown [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+In a previous life I spent most of my time (free and otherwise) thinking about
+phonetics and the analysis of speech sounds. Like most scientific research, it
+was slow and tedious and oddly rewarding.
+
+Back then I was already in a steady drift towards spending more time playing
+with computers than with my research, so a lot of my efforts went into
+studying [Praat](https://www.fon.hum.uva.nl/praat), a piece of software you've
+almost certainly never heard of, but that has probably been used in most of
+the phonetics research you've also never heard of.
+
```
Comment (mohawk2) [inline statocles-site/blog/2024/12/16/pitch-detection/index.markdown:31]: Could you insert a section break `---` here, so the RSS feed summary (and `blog/` page) are nice and short? Creates interest that will force any reasonable person to read the rest :-)

File: statocles-site/blog/2024/12/16/pitch-detection/index.markdown [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+to these windows of analysis as "frames".
+
+This article (which is based on [a similar one] I wrote in 2012 using R) will
+go over the core part of the algorithm as described in [Boersma (1993)], and
+since this is the PDL Advent Calendar, we'll be using PDL for illustrating
+each step and generating the figures. I'll be showing the relevant bits of
+code at each step, but you can also download [a file with all of the code] in
+context.
+
+[boersma (1993)]: http://www.fon.hum.uva.nl/paul/papers/Proceedings_1993.pdf
```
Comment (mohawk2) [inline statocles-site/blog/2024/12/16/pitch-detection/index.markdown:63]: Can you confirm you've checked Statocles likes this formatting? See my notes on #38 for how to iterate.

**Review by mohawk2 — CHANGES_REQUESTED** [review]
This looks really good! Can you deal with the two tiny points, and also change it to go in on the 15th not 16th?

File: statocles-site/blog/2024/12/16/pitch-detection/index.markdown (hunk 1 above)
Comment (jjatria) [reply statocles-site/blog/2024/12/16/pitch-detection/index.markdown]: Done!

File: statocles-site/blog/2024/12/16/pitch-detection/index.markdown (hunk 2 above)
Comment (jjatria) [reply statocles-site/blog/2024/12/16/pitch-detection/index.markdown]: You mean the links with the implicit references, etc? Yeah, I confirmed it was working.

Incidentally, this is the command I was using to try it, which means it didn't mess with the directory I was currently in:

```
pushd $( git rev-parse --show-toplevel ); (cd statocles-site/; rm -rf .statocles; statocles build --date 2024-12-30); rm -rf advent/; cp -r statocles-site/.statocles/build/ advent/; popd
```

**Comment by jjatria** [comment]
Done and done!

**Comment by mohawk2** [comment]
Thank you!

**Pull request merged**